### ISSUE TITLE
Music playing in some locations when volume is set to "off"

### ISSUE THREAD

**Issue by Winddbourne** [OreCruncher/DynamicSurroundings#732]
**Mod Version:**

1.16.4-4.0.4.2

**Forge Version:**

36.1.16

**Description:**

I've got my music volume set to "off" but for some reason certain locations have music playing anyway. It just happened in a dark forest biome, and it seems to be extremely location specific. If I move a bit away the music stops, move back and it starts again. 

I simply have no idea what is causing this. I see something called "spot sounds" in the config, and it has it's own volume could that be what is happening here?  

I'd like to stop it because I'm extremely sensitive to sound. That's why I turn the music off to begin with. I don't want it playing.  I'm hoping you have some idea what's going on.

**Comment by OreCruncher** [comment]
DS does not have any music tracks in 1.16.x.  When you say music do you really mean music vs. some other sounds?

**Comment by Winddbourne** [comment]
Yes I mean actual music and I can't figure out why it would start playing. There isn't anything special there. It's just a random patch of forest. It's driving me nuts. 

This happens in both my Nightfall mod pack and this new one I'm working on and nothing shows up in the logs except that the sound manager is sometimes lagging. That sort of made sense, a new sound manager from D.S could be doing something so I started looking at the config files for anything that even might relate to the game music.  

I couldn't find anything so I figured I'd check here. It's certainly not game breaking but it is so annoying. lol
